src/procedures/enable-multiprotocol.html

```diff
@@ -0,0 +1,18 @@
+---
+layout: guide
+permalink: "guides/{{ page.fileSlug }}/index.html"
+description: Home Assistant Yellow
+guideName: Enable Multiprotocol support
+nvme: false
+collectionName: enable-multiprotocol
+tools:
+  title: Prerequisites
+  image: yellow-multiprotocol-02.png
+  content: |
+    <p>This guide assumes you have already completed the following step:</p>
+    <ul>
+      <li>Have the Zigbee Home Automation (ZHA) integration up and running.</li>
```

Comment:
The ZHA integration is enabled automatically on HAOS. So we don't really need a prerequisite here.

Maybe just saying "Have the Zigbee Home Automation (ZHA) integration up and running." still make sense, just so its clear that this part should be working.